backend/src/v5/processors/teamspaces/projects/models/containers.js

```diff
@@ -80,17 +80,15 @@ Containers.getContainerStats = async (teamspace, container) => {
 	return stats;
 };
 
+Containers.getRevisionFormat = (rFile) => (rFile ? '.'.concat(rFile[0].split('_').pop()) : undefined);
```

Comment:
This should live in `models/revisions.js`, we're manipulating a data that belongs to revision rather than general container